src/sage/modules/free_quadratic_module_integer_symmetric.py

```diff
@@ -1535,7 +1535,8 @@ def short_vectors(self, n, **kwargs):
         from sage.quadratic_forms.quadratic_form import QuadraticForm
         q = QuadraticForm(e * self.gram_matrix())
         short = q.short_vector_list_up_to_length(n, **kwargs)
-        return [[self(v * self.basis_matrix()) for v in L] for L in short]
+        # (matrix(L)).rows() * B is faster than [v * B for v in L]
```

Comment:
```suggestion
        # (matrix(L)* B ).rows() is faster than [v * B for v in L]
```
I think this is what you meant.